README.md

```diff
@@ -658,7 +658,7 @@ Send a Pull Request with additions, modifications or removals. For new contribut
 Lists companies known to use popular ROS stacks.
 
 #### Navigation
-Lists companies known to use the ROS Navigation stack.
+Lists companies known to use the ROS 2's Nav2 framework.
```

Comment:
```suggestion
Lists companies known to use the ROS Navigation stack and ROS 2's Nav2 framework.
```

Reply:
Just echoing in here as well @SteveMacenski,

> list using both ROS Navigation stack and ROS 2 Nav2 framework. Tried summarizing both as "ROS Navigation stack" but it seems that doesn't work.
>
> Left a suggestion on top of your comment. Let me know what works best and/or if there's a better way (shorter) to refer to both.

I don't mind having separately named but I'd prefer a unified one when referring to both. Any thoughts on this?

Reply:
Yeah that suggested change sounds good to me.